app/models/katello/host/content_facet.rb

```diff
@@ -19,6 +20,7 @@ class ContentFacet < Katello::Model
 
       validates :content_view, :presence => true, :allow_blank => false
       validates :lifecycle_environment, :presence => true, :allow_blank => false
+      validates :content_source, :presence => true, :allow_blank => false
```

Comment:
This wasn't ever required before.  I think it may have quite a few implications to requiring it.  Let me think about this for a bit